InformationScripting/src/nodes/CompositeQueryNode.h

```diff
@@ -39,6 +39,9 @@ class INFORMATIONSCRIPTING_API CompositeQueryNode : public Super<QueryNode>
 {
 	COMPOSITENODE_DECLARE_STANDARD_METHODS(CompositeQueryNode)
 	ATTRIBUTE(Model::TypedList<QueryNode>, queries, setQueries)
+
+	public:
```

Comment:
Actually, yes, I also forgot about it during our meeting. Please use the visitor. It will simplify the code in the nodes.